spec/shoryuken/queue_spec.rb

```diff
@@ -86,12 +86,14 @@
       expect(sqs).to receive(:send_message).with(hash_including(message_body: 'msg1'))
 
       subject.send_message(message_body: 'msg1')
+      sleep(0.0001)
```

Comment:
as per my PR message - I need to do this in more specs to get them all of them to pass